docs/guides/headless-cms/build-static-website/next-13.md

```diff
@@ -230,15 +231,16 @@ import directus from 'lib/directus';
 import { notFound } from 'next/navigation';
 
 async function getPost(slug) {
-	try {
-		const post = await directus.items('posts').readOne(slug, {
-			fields: ['*.*'],
-		});
-
-		return post;
-	} catch (error) {
-		notFound();
-	}
+  try {
+    const post = await directus.request(
+      readItem('posts', slug, {
+        fields: ['*.*'],
```

Comment:
Nested wildcards are no longer supported in the SDK. You can do the following with explicit relations but not chaining wildcards
```
fields: ['*', { 'relation': ['*'] }],
```